test/sequential/test-dgram-bind-shared-ports.js

```diff
@@ -96,7 +96,7 @@ if (cluster.isPrimary) {
       common.mustCall((err) => {
         process.send(`socket3:${err.code}`);
       });
-  const address = common.localhostIPv4;
+  const address = common.localhostIP;
```

Comment:
We need to keep this to IPv4, or to change the sockets to `udp6`:

https://github.com/nodejs/node/blob/117e293c376d54665e3c7d7b5b2ed15ef530ada1/test/sequential/test-dgram-bind-shared-ports.js#L76-L78

```js
  const type = common.hasIPv6 ? 'udp6' : 'udp4';
  const socket1 = dgram.createSocket(type, common.mustNotCall());
  const socket2 = dgram.createSocket(type, common.mustNotCall());
  const socket3 = dgram.createSocket(type, common.mustNotCall());
```